packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/description.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+
+type TDescription = {
+    id: string;
+    description: string;
+};
+
+const Description = ({ id, description }: TDescription) => (
+    <React.Fragment>
+        {id === 'type-strategy' && (
+            <div key='description' className='quick-strategy__text'>
+                <Text text='sm' weight='normal'>
```

Comment:
no need to specify normal default weight is normal

Reply:
and there is no `text` property supported by Text Component, I believe you wanted to set `size='sm'`? @maryiafrantsava-binary

Reply:
@sandeep-deriv thanks that you noticed, actually it is an unnecessary attribute